graphql_compiler/cost_estimation/statistics.py

```diff
@@ -82,13 +82,28 @@ def get_distinct_field_values_count(self, vertex_name, field_name):
         """
         return None
 
+    def get_field_quantiles(self, vertex_name, field_name):
+        """Return a list dividing the field values in equally-sized groups.
+
+        Args:
+            vertex_name: str, name of a vertex defined in the GraphQL schema.
+            field_name: str, name of a vertex field. This field has to have a type on which
```

Comment:
nit: `name of a property field on that vertex type` perhaps, to avoid ambiguity relative to the existing `vertex field` definition in the spec?